notification-portlet-webapp/src/main/webapp/WEB-INF/jsp/modal.jsp

```diff
@@ -55,7 +55,7 @@ recommended and more common approach).
 
 <div id="${n}" class="modal fade" role="dialog" data-backdrop="static">
     <div class="modal-dialog">
-        <div class="modal-content np-content">
+        <div class="modal-content np-content" tabindex="-1">
```

Comment:
Should the `tabindex` be on line 56?
Similar to the Bootstrap docs: https://getbootstrap.com/docs/3.3/javascript/#modals

Reply:
It looks like it will work on line 56. I'll update shortly.